Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -68,37 +187,142 @@ public class Functions
 }
 ```
 
-and place in your template:
+```json
+    "CloudCalculatorFunctionsAddGenerated": {
+      "Type": "AWS::Serverless::Function",
+      "Metadata": {
+        "Tool": "Amazon.Lambda.Annotations",
+        "SyncedEvents": [
+          "RootGet"
+        ]
+      },
+      "Properties": {
+        "Runtime": "dotnet6",
+
+...
+        "Role": {
+          "Fn::GetAtt": [
+            "LambdaRoleParameter",
+            "Arn"
+          ]
+        }
+      }
+    },
+```
+
+## Amazon API Gateway example
+
+This example creates a REST API through Amazon API Gateway that exposes the common arithmetic operations. 
+
+To avoid putting business logic inside the REST API a separate calculator service is created to encapsulate the logic of the arithmetic operations. Here is both the 
+calculator service's interface and default implementation.
+
+```csharp
+public interface ICalculatorService
+{
+    int Add(int x, int y);
+
+    int Substract(int x, int y);
```

Comment:
Typo in "Subtract" throughout both code blocks.

Reply:
Fixed. Something about that word always messes me up.

Reply:
Still 4 more instances of `substract` in the next codeblock with the function implementations

Reply:
Fixed